tests/macros.rs

```diff
@@ -103,3 +121,26 @@ macro_rules! at_and_ucmd {
         (ts.fixtures.clone(), ts.ucmd())
     }};
 }
+
+#[macro_export]
+macro_rules! fixtures_path {
+    ($filename:expr) => {{
+        let mut fixtures_path = fixtures_dir!();
+        fixtures_path.push($filename);
+        fixtures_path
+    }};
+}
+
+#[macro_export]
+macro_rules! pred_eq_file {
+    ($filename:expr) => {
+        predicate::path::eq_file(fixtures_path!($filename).as_path())
+    }
+}
+
+#[macro_export]
+macro_rules! pred_str_contains {
+    ($str:expr) => {
+        predicate::str::contains($str).from_utf8()
```

Comment:
RE `from_utf8`

Right now, the only automatic conversions that happen in our API are [u8 slice to predicate and str to predicate](https://github.com/assert-rs/assert_cmd/blob/master/src/assert.rs#L382).

Feel free to open an issue about converting `Predicate<str>` into `Predicate<[u8]>`. or any other ideas to simplify the use of the API.

Reply:
Yes, this one bother me a lot. I think it could work, but realize it doesn't when compile the code. It took me some time to find out the reason because the documentation of `assert_cmd` and `predicates` are mixed. I should check with these two docs.

Reply:
Could you open issues on any documentation problems you came across?  We're wanting to make sure this is accessible for people starting a first Rust project to create a CLI.  My challenge is the curse of knowledge: I'm too familiar with it to see know what pitfalls a new user will run into.